ubiquity/dell-bootstrap.py

```diff
@@ -760,6 +761,16 @@ def prepare(self, unfiltered=False):
             self.log("rec_type %s, stage %d, device %s" % (rec_type, self.stage, self.device))
             if (rec_type == 'factory' and self.stage == 2) or rec_type == 'hdd':
                 self.fixup_factory_devices(rec_part)
+            if rec_type == 'hdd':
+                # copy old mok key so that user don't need to enroll it again.
+                rootfs = mount[0:-1] + '3'
```

Comment:
If using a hardcoded value, should at least use the define `EFI_OS_PARTITION`.

Reply:
thanks for the reminder, will change this part/